### DISCUSSION TITLE
how do I  extract a .exe from the software developers installation web page?

### DISCUSSION THREAD

**Discussion by ineedtechhelp** [community/community#141491] [Programming Help]
### Select Topic Area

Question

### Body

sorry this is long, but the first couple sentences give you the gist of this, but further detail afterwards if needed:  downloaded & Installed Winlator On my android tablet So that I could run " The Latest windows version" of An application called "e-sword".  There IS an android version of this application, But it does not do what the windows version does.  That is The reason I need to install the windows version of E-sword On my android tablet.  
     The software publisher does not provide a .exe file Of their application for me to Download & Copy into a Winlator Container And then run it.  The software publisher Only provides a link to click on their website that will install their application  onto a person's windows p c. Maybe there is a .exe File of their latest version, But I have googled and searched for weeks, And I have now found it.
     The website of this software developer is  www.e-sword.net.  There you will see the link to install their application onto a windows p c.  That's it.  There is no .exe Of their program available That I can find, so that I could then copy it into a Winlator Container, and then run it from within the container. ( I am totally new to Winlator & github. Sorry If i'm not Using the right terminology or not doing something correctly.  Will someone please help me get E-sword installed into a Winlator container ( If that's the correct terminology to use)?
    Thank you in advance for Helping me. Getting the WINDOWS  version of E-sword running on my Android Tablet Is the primary reason for me, purchasing this expensive tablet ( Although I got it for a bargain price, from bestBuy's open box sales link).  My tablet is a samsung galaxy S8 Ultra running the latest version of android.

https://www.e-sword.net/files/e-sword_1410_setup.exe

Where do I get the .exe Of a windows program That is only available through a setup link.Find the software publisher's website?

GITHUB POST: I downloaded & Installed Winlator On my android tablet So that I could run " The Latest windows version" of An application called "e-sword".  There IS an android version of this application, But it does not do what the windows version does.  That is The reason I need to install the windows version of E-sword On my android tablet.  
     The software publisher does not provide a .exe file Of their application for me to Download & Copy into a Winlator Container And then run it.  The software publisher Only provides a link to click on their website that will install their application  onto a person's windows p c. 
     The website of this software developer is  www.e-sword.net.  There you will see the link to install their application onto a windows p c.  That's it.  There is no .exe Of their program available That I can find, so that I could then copy it into a Winlator Container, and then run it from within the container. ( I am totally new to Winlator & github. Sorry If i'm not Using the right terminology or not doing something correctly.  Will someone please help me get E-sword installed into a Winlator container ( If that's the correct terminology to use)?
    Thank you in advance for Helping me. Getting the WINDOWS  version of E-sword running on my Android Tablet Is the primary reason for me, purchasing this expensive tablet ( Although I got it for a bargain price, both best. Buy open box sales).  My tablet is a samsung galaxy S8 Ultra running the latest version of android.

**Comment by tpmst** [comment]
Hello @ineedtechhelp,

I understand that you're trying to install the Windows version of e-Sword on your Samsung Galaxy Tab S8 Ultra using Winlator. You might have misconfigured your Winlator Container.

Open Winlator:
 - Launch the Winlator app on your tablet.

Create Container:
 - Tap the "+" button to create a new container.
 - Name your container (e.g., "e-Sword").
 - Select the architecture (usually x86 for 32-bit apps or x86_64 for 64-bit apps). e-Sword is a 32-bit application, so choose x86.
    Configure Settings:

Leave default settings unless specific configurations are needed.
Save the container.

Using a File Manager:
 - Open your preferred file manager app on your tablet.
 - Navigate to the folder where e-sword_1410_setup.exe is stored.
 - Copy the installer file.

Navigate to the Container Directory:
 - Go to Internal Storage/Android/data/com.winlator/files/containers/[YourContainerName]/drive_c/.
 - You may need to enable "Show hidden files" in your file manager settings to access these folders.

Paste the Installer:
 - Paste e-sword_1410_setup.exe into the drive_c directory or create a new folder within it (e.g., "Installers") and paste it there.

Launch Winlator:
 - Open the Winlator app.
 - Select your "e-Sword" container.

Run the Installer:
 - Tap on "Run EXE" or a similar option.
 - Navigate to where you pasted the installer (drive_c or the folder you created).
 - Select e-sword_1410_setup.exe.

Install e-Sword:
 - The e-Sword installer should launch.
 - Follow the on-screen instructions as you would on a Windows PC.
 - Choose installation directories (default paths are usually fine).
